mne/viz/tests/test_raw.py

```diff
@@ -326,7 +326,7 @@ def test_plot_raw_selection(raw, browser_backend):
         # across platforms.
         # (QTest.mouseClick works isolated on all platforms but somehow
         # not in this context. _fake_click isn't working on linux)
-        sel_fig._chkbx_changed(list(sel_fig.chkbxs.keys())[0])
+        sel_fig._chkbx_changed(None, list(sel_fig.chkbxs.keys())[0])
```

Comment:
The tests are run with the stable version of `mne-qt-browser`, not with main?

Reply:
I can't remember, but regardless of how we actually run the tests, we should maintain backward compat with versions where possible. Imagine some dev having mne-qt-browser 0.3 or 0.4 (or even 0.5.dev0 before your open PR), they update MNE-Python, then all of a sudden all mne-qt-browser tests are failing and we've wasted their time as they try to figure out they need to upgrade...

Reply:
Good point, thanks I did not think about the dev use case. 
If you don't mind, I prefer to add a check on the method signature here rather than swapping arguments around inside the method. I think:

```
if label is None:
    label = checked
    checked = True
```

will be hard to re-read and understand later.

Reply:
The problem with the check on the method signature is that if someone else is on MNE (stable/released or main) before this PR and then updates to mne-qt-browser `main` (after that PR lands) then again suddenly all tests fail for them...

I think a suitable comment can make this logic understandable

Reply:
Arf the other way around breaks too..
It took me more than one read to wrap my head around it, I'm not used to this kind of logic between 2 repositories using tests from both repositories 😓

Reply:
Yes it's a pain but ideally if one has to adapt it will be `mne-qt-browser`. In a worst case over there we can go to `*args, *, checked=None, label=None` and parse the intent...